src/CommentCard.jsx

```diff
@@ -1,11 +1,45 @@
-export default function CommentCard({ comment }) {
-  const { body, author, created_at } = comment;
-
-  return (
-    <div className="comment-card">
-      <p>{body}</p>
-      <p>comment by: {author}</p>
-      <p>created at: {created_at.slice(0, 10)}</p>
-    </div>
-  );
+import { useState, useEffect } from "react";
+import deleteCommentByID from "./Utils/deleteCommentByID";
+
+export default function CommentCard({ comment, commentList, setCommentList }) {
+  const [deleteComment, setDeleteComment] = useState(false);
+  const [hasDeleted, setHasDeleted] = useState(false);
+  const { body, author, created_at, comment_id } = comment;
+  const condition = author === "grumpy19";
+
+  useEffect(() => {
+    if (deleteComment === true) {
+      deleteCommentByID(comment_id).then(() => {
+        setDeleteComment(false);
+      });
+    }
```

Comment:
Currently if the api request to delete the comment fails (for example when the internet cuts out), the user isn't given any indication that the comment hasn't actually been deleted. Can you add some error handling so it's clear to the user when deletion has failed 🔍